src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -54,87 +63,75 @@ const defaultProps = {
     bankAccountList: [],
     cardList: [],
     isLoadingPayments: false,
+    enableSelection: false,
+    selectedAccountID: '',
+    filterType: '',
     isAddPaymentMenuActive: false,
 };
 
 class PaymentMethodList extends Component {
     constructor(props) {
         super(props);
-
+        this.state = {
+            selectedAccountID: this.props.selectedAccountID,
```

Comment:
We should use the value from props? It's usually the wrong pattern to initialize state with props.